packages/trader/src/Assets/SvgComponents/launch/ic-chart-launch.svg

```diff
@@ -0,0 +1,62 @@
+<svg width="216" height="128" viewBox="0 0 216 128" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Needs to be minimized. SVG is only one color instead of themed?

Reply:
@ahmadtaimoor-deriv please run svgo on this file to minimize it. There's an instruction how to use it in the readme I shared last time.